parachain/primitives/ethereum/src/lib.rs

```diff
@@ -386,4 +386,30 @@ mod tests {
 		];
 		assert!(header.check_receipt_proof(&proof_receipt263).is_some());
 	}
+
+	#[test]
+	fn serde_to_string() {
+		let header = Header {
+			parent_hash: hex!("fe69ca8944fb5a147cf975d28853a91a9ee6c4d4a87ba5e1a1807814a8ecfb84").into(),
+			timestamp: 1613067619u64.into(),
+			number: 9643680u64.into(),
+			author: hex!("56a8e9614f19e46c7c2fe0cfca217ca65e960a60").into(),
+			transactions_root: hex!("dfd8f5014d55e9dd9c026e63f3b62c6afb5a3e33c42f3651f0466d21e4e318c9").into(),
+			ommers_hash: hex!("1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347").into(),
+			extra_data: hex!("d883010919846765746888676f312e31352e36856c696e7578").into(),
+			state_root: hex!("1161f9af771c5c8072ffdca1f7d9263bb1e83e89d277543de088979c8fdf48e2").into(),
+			receipts_root: hex!("18dd1c1726803a6f2c7bd3ec01ce784ac0736718adba2c180ab717b7d674ef4c").into(),
+			logs_bloom: (&hex!("00000000000000040000002000000000000000010000200000800000000004000000400000000000100000000000000000000000000000000000000000000000000000400000000000000008000000000001800000000000000104000000000000000000120000000000000000000800000000000020000000000010000000400200000000000000000000000000000000000000000000000800000000800000000000000000000400000004000000000000004000000000000002000000000000000002000000000000000000000000000000001000000000000000000020000000000000000000000000008000000000000000000000000000008000000400")).into(),
+			gas_used: 4335047u64.into(),
+			gas_limit: 8000000u64.into(),
+			difficulty: 1026821511u64.into(),
+			seal: vec![
+					hex!("a0da5df4f798873fbe974ee43e2f6ee2dcf44a42dfb127643c954a8dca953710ac").to_vec(),
+					hex!("88016e536fc135f42d").to_vec(),
+			],
+		};
+
+		let j = serde_json::to_string(&header).unwrap();
+		println!("{}", j);
+	}
```

Comment:
Probably not relevant because this is a draft, but leaving a comment here anyways to potentially remove this :)